<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLyneWindow</class>
 <widget class="QDialog" name="EditLyneWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>18</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>FreeSerif</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <italic>true</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Add / Edit Lyne</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="editGroupBox">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>25</y>
     <width>301</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Business Information:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="objectIdBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>34</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Object ID</string>
    </property>
    <widget class="QLineEdit" name="objectIdLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="businessNameBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>87</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Business Name</string>
    </property>
    <widget class="QLineEdit" name="businessNameLineEdit">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="regionBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Region</string>
    </property>
    <widget class="QLineEdit" name="regionLineEdit">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="businessTypeBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>192</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Business Type</string>
    </property>
    <widget class="QLineEdit" name="businessTypeLineEdit">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="addressBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Business Address</string>
    </property>
    <widget class="QLineEdit" name="businessAddressLineEdit">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="businessPhoneBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Business Phone Number</string>
    </property>
    <widget class="QLineEdit" name="businessPhoneLineEdit">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="selectablesBox">
   <property name="geometry">
    <rect>
     <x>583</x>
     <y>416</y>
     <width>381</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Stream Settings:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="camTypeBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>23</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera Type</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="camTypeComboBox">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>24</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>23</y>
      <width>120</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Number of Cams</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="camUrlBox">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>74</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Camera URL</string>
    </property>
    <widget class="QLineEdit" name="camUrlLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>24</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="processIdBox">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>123</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Process ID</string>
    </property>
    <widget class="QLineEdit" name="processIdLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>24</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="streamActiveBox">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>22</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Enabled</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QCheckBox" name="streamActiveCheckBox">
     <property name="geometry">
      <rect>
       <x>22</x>
       <y>26</y>
       <width>20</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="hoursGroupBox">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>416</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Hours (24 hr clock)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="openLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>111</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="closeLabel">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>151</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mondayCheckBox">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mon</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mondayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mondayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tuesdayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tuesdayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="tuesdayCheckBox">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tues</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wednesdayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="wednesdayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>194</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="mondayCheckBox_3">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wed</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="thursdayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="thursdayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>259</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="thursdayCheckBox">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Thur</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fridayCheckBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fri</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fridayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="fridayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="saturdayCheckBox">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saturdayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="saturdayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>389</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="sundayOpenLineEdit">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>108</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="sundayCheckBox">
    <property name="geometry">
     <rect>
      <x>458</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sun</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sundayCloseLineEdit">
    <property name="geometry">
     <rect>
      <x>454</x>
      <y>147</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>36</y>
      <width>79</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timeZoneLabel">
    <property name="geometry">
     <rect>
      <x>107</x>
      <y>39</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Zone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>41</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UTC Offset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="utcOffsetLineEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>38</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>63</y>
      <width>521</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>593</x>
     <y>25</y>
     <width>361</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Business Specials:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QGroupBox" name="special1Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Special 1:</string>
    </property>
    <widget class="QLineEdit" name="special1LineEdit">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>23</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="special2Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>111</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Special 2:</string>
    </property>
    <widget class="QLineEdit" name="special2LineEdit">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>23</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="special3Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Special 3:</string>
    </property>
    <widget class="QLineEdit" name="special3LineEdit">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>23</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="special4Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>268</y>
      <width>341</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Special 4:</string>
    </property>
    <widget class="QLineEdit" name="special4LineEdit">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>23</y>
       <width>331</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="buttonsBox">
   <property name="geometry">
    <rect>
     <x>371</x>
     <y>89</y>
     <width>191</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>112</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>55</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>169</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
